sql/parser/eval.go

```diff
@@ -441,15 +589,27 @@ var binOps = map[binArgs]func(Datum, Datum) (Datum, error){
 
 	// TODO(pmattis): Overflow/underflow checks?
 
-	// TODO(pmattis): Should we allow the implicit conversion from int to float
```

Comment:
Ah, this comment is ambiguous. I was thinking it was referring to the (int, int) case which still casts to floats. I guess this is OK to remove.